<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please sign in</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>passwd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>result</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="resultTextBrowser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>resultTextBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>passwdLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>nameLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
